tests/lib/devicetree/api/app.overlay

```diff
@@ -115,6 +115,7 @@
 			#gpio-cells = < 0x2 >;
 			#foo-cells = < 0x1 >;
 			label = "TEST_GPIO_1";
+			zephyr,linker-region = "TEST_REGION";
```

Comment:
Putting this in a GPIO node is a weird choice. Why not just create a fake mmio-sram node or something like that?

Reply:
Added to `mmio-sram` node in new test application.